src/Run/Setup.cs

```diff
@@ -77,40 +121,83 @@ private string FindSettingType(string valueToFind)
             return null;
         }
 
-        public void prepareValues(string os, Dictionary<string, string> parameters, string configFile)
+        public void PrepareValues(string os, Dictionary<string, string> parameters, string configFile)
         {
             SettingParameters = new Dictionary<string, string>(parameters);
             Os = os;
             ConfigurationFilePath = configFile;
+            // Add defaults for run tool settings if they haven't been define din config.json
+            SetRunToolSettingsDefaults();
+            ToolSettings = Settings.Where(s => s.Value.ValueType == null ||
```

Comment:
@maririos pointed out that there is no current validation that these ToolSettings are actually properly formed, and someone could add a new setting but leave out the ValueType, so then we would think it's a "run tool setting".  I'll add validation to ensure that anything which is not a run tool setting has proper values set for "ValueType".